feat🍄 : Add start section components

twkimfe opened the pull request:
- Added atomic, molecular, organism unit of start section
- finished test

## PR 유형
- [x] 새로운 기능 추가
- [x] CSS 등 사용자 UI 디자인 변경

## PR 체크리스트
<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->
- [x] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [x] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### PR 상세
start 섹션, 원자 단위, 분자 단위, 유기체 단위 컴포넌트 완성

## 이슈
resolves #4

twkimfe committed:
feat🍄 : Add start section components
- Added atomic, molecular, organism unit of start section
- finished test

hyunji1117 commented on src/styles/component-start.css:
```diff
@@ -52,9 +51,213 @@
+}
+
+/* Common/button CSS */
+.component-organism-list-start-button {
```
class명 조직체명에 맞게 수정 부탁드립니다~

twkimfe replied:
넵 알겠습니다~!

hyunji1117 commented on src/styles/component-start.css:
```diff
@@ -52,9 +51,213 @@
+  }
+}
+
+.component-organism-start-nav--list-item {
```
html 내 변경된 class와 동일하게 수정 반영하시면 될 것 같아요~!

Flowyson reviewed (changes requested):
태우님 클래스명 확인 부탁드립니다

twkimfe committed:
refactor♻️: Modify component-start class name
- Modified component-start.html class name
- Modified component-start.css class name

hyunji1117 reviewed (approved):
태우님 rem 단위 하나씩 수정하느라 고생많으셨습니다!ㅎㅎ👍

Flowyson reviewed (approved):
태우님 클래스명 수정 확인 완료했습니다
